### ISSUE TITLE
Do you considering making a new tab grouper for Firefox?

### ISSUE THREAD

**Issue by vgranger1989** [starsolaris/tab_grouper#6]
Hello Andrey! 
I've been using Opera 12 for 10 years, and Tab Grouper is my favorite extension(*´∀｀)
But opera was too slow now. I turned to Firefox, but found no tab group extension better than yours!
So do you considering coding a new one for Firefox?
Thank you very much! Спасибо!

**Comment by starsolaris** [comment]
I can try, but i thought that Firefox and Chrome have many other extensions like this.

**Comment by vgranger1989** [comment]
Yes there are, but I tried several extensions and found no one was as good as tab grouper - some extensions do the sort job, and some others do the group job, while Tab Grouper does both. 
But this is only my advice, anyway, thank you very much for creating Tab Grouper!
